src/OpenTelemetry.Exporter.Ocagent/Implementation/SpanDataExtentions.cs

```diff
@@ -119,13 +119,14 @@ private static Span.Types.TimeEvents FromITimeEvents(ITimedEvents<IEvent> events
 
         private static Span.Types.Link FromILink(ILink source)
         {
-            return new Span.Types.Link
+            var result = new Span.Types.Link
             {
-                TraceId = ByteString.CopyFrom(source.TraceId.Bytes),
-                SpanId = ByteString.CopyFrom(source.SpanId.Bytes),
-                Type = source.Type == LinkType.ChildLinkedSpan ? Span.Types.Link.Types.Type.ChildLinkedSpan : Span.Types.Link.Types.Type.ParentLinkedSpan,
                 Attributes = FromIAttributeMap(source.Attributes),
+                TraceId = ByteString.CopyFrom(source.Context.TraceId.Bytes),
+                SpanId = ByteString.CopyFrom(source.Context.SpanId.Bytes),
```

Comment:
is it possible already to add tracestate here or we need to recompile proto files? It's already defined in https://github.com/open-telemetry/opentelemetry-java/blob/ca0ec9dfa83fd8c4b1a9e4a5af3db75cf64c6cbc/sdk/src/main/proto/trace/v1/trace.proto#L215

Are we tracking it somewhere?

Reply:
there is an issue to update protos